src/main/java/past/cwdt/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package past.cwdt.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotBlank @Email String email,
```

Comment:
```suggestion
        @NotBlank @Email(regexp = "Your custom regex") String email,
```
by default, the email annotation is checked against the regular expression ".*". That is, any string is valid. Therefore, I would recommend adding any regular expression (because now this annotation does not validate at all)